### PR TITLE
storage: Use the Mintlayer fork of the `lmdb` crate

### PR DESCRIPTION
**Pull request by iljakuklic**

### CODE REVIEW

**Review by TheQuantumPhysicist — APPROVED** [review]

File: storage/lmdb/Cargo.toml [hunk 1]
```diff
@@ -8,7 +8,9 @@ license = "MIT"
 storage-core = { path = '../core' }
 utils = { path = '../../utils' }
 
-lmdb-rkv = "0.14"
+[dependencies.lmdb-mintlayer]
```
Comment (TheQuantumPhysicist) [inline storage/lmdb/Cargo.toml:11]: Why not use braces like others?

**Review by muursh — APPROVED** [review]

File: storage/lmdb/Cargo.toml (hunk 1 above)
Comment (iljakuklic) [reply storage/lmdb/Cargo.toml:11]: I just found the line too long but can change it back, no strong opinions here

Comment (stanislav-tkach) [reply storage/lmdb/Cargo.toml:11]: Personally I prefer the "single line" dependency format, but in the p2p we have a lot of dependencies in the "multiline" format.

**Review by stanislav-tkach — APPROVED** [review]

File: storage/lmdb/Cargo.toml (hunk 1 above)
Comment (TheQuantumPhysicist) [reply storage/lmdb/Cargo.toml:11]: Would be great if you could change it... it makes global searches much easier when everything is homogeneous. I want to change them also in p2p.

Comment (iljakuklic) [reply storage/lmdb/Cargo.toml:11]: ok

**Pull request merged**